src/KafkaNET.Library/Requests/FetchRequest.cs

```diff
@@ -56,11 +56,11 @@ public class FetchRequest : AbstractRequest, IWritable
         public const byte DefaultMinBytesSize = 4;
         public const byte DefaultOffsetInfoSizeSize = 4;
 
-        public const short CurrentVersion = 1;
```

Comment:
It looks like since I started working on this that this was changed from 0 to 1. I have reverted it to zero again in order to ensure backwards compatibility, since the previous code did not properly parse ThrottleTime (which is included in v1 fetch responses.)